frontend/server/src/main/java/com/amazonaws/ml/mms/http/HttpRequestHandler.java

```diff
@@ -58,6 +58,9 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpRequest req) {
         } catch (BadRequestException | ModelException e) {
             logger.trace("", e);
             NettyUtils.sendError(ctx, HttpResponseStatus.BAD_REQUEST, e);
+        } catch (ConflictStatusException e) {
```

Comment:
You might want to add a test as well [ModelServerTest.java](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/test/java/com/amazonaws/ml/mms/ModelServerTest.java). Also, update openapi spec ([management_open_api.json](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/test/resources/management_open_api.json#L825) and [OpenApiUtils.java](https://github.com/awslabs/mxnet-model-server/blob/master/frontend/server/src/main/java/com/amazonaws/ml/mms/openapi/OpenApiUtils.java#L387)) to reflect this change.

Reply:
Will do, thanks!